src/App/App.js

```diff
@@ -1,35 +1,47 @@
 import './App.scss';
 import Nav from '../Nav/Nav';
 import React, { Component } from 'react';
-import movieData from '../movieData';
+//import movieData from '../movieData';
 import MovieContainer from '../MovieContainer/MovieContainer';
 import Movie from '../Movie/Movie';
+import { callApi, callSingleApi } from '../callApis'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
-      movieSelected: {} 
+      movies: [],
+      movieSelected: {},
+      error: null
     }
   }
+  //integration test, tests the methods
+  handleClick = (id) => {
+    //fetch her for single movie
+    // .then(data => this.setState({movieSelected: data}))
+    callSingleApi(id)
+      .then(data => this.setState({ movieSelected: data.movie }))
+      .catch(error => this.setState({ error }))
```

Comment:
You're setting the state to a full Error object. That may be what you want, but it's more likely you want the `message` property of the Error. Also, make sure that you're _unsetting_ this error at some point; otherwise, the user may end up seeing a stale error message in the future.